Woof/Woof/Model/PetOwner.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+/**
+ A struct representing a pet owner.
+ This struct conforms to the Identifiable and Codable protocols, allowing it
+ to be easily used in SwiftUI and encoded/decoded from JSON.
+ */
+struct PetOwner: Identifiable, Codable {
+    /// Uniquely identifies the pet owner.
+    let id: UUID
+    
+    /// The first name of the pet owner.
+    var name: String
+    
+    /// The last name of the pet owner.
+    var surname: String
+    
+    /// The phone number of the pet owner.
+    var phone: String
+    
+    /// The optionl URL of the pet owner's avatar image.
```

Comment:
nit: I see it's optional :) 

```suggestion
    /// URL that contains an avatar image of the pet owner.
```

Reply:
Ok, i'll fix it